### ISSUE TITLE
Cannot navigate to Service Page

### ISSUE THREAD

**Issue by eitansuez** [kiali/kiali#4918]
Hello, I am trying to work through the kiali travel demo tutorial ([travel](https://kiali.io/docs/tutorials/travels/03-first-steps/).
i am standing up a gke cluster, following the instructions, installed istio 1.13.2 demo profile, applied the jaeger, prometheus and grafana addons, then installed kiali per the instructions here: https://kiali.io/docs/tutorials/travels/01-prerequisites/ using helm.
i deployed the travel app.  turned on sidecar injection for travel-control.  i'm on the step where i need to navigate to the control service in the travel-control namespace to use the request routing wizard to generate a traffic rule.

1. when i navigate to "services" i notice the "no health information" icon on the control service.
2. when i navigate to the control service, the page displays "Sorry, there was a problem. Try a refresh or navigate to a different page."  the chrome devtools console shows "TypeError: Cannot read properties of null (reading 'map')" at react_devtools_backend.js:3973

here's the kiali version from the 'about' page:

Kiali UI 1.48.0 (cbe6de73af0a34966f897e805f4dc74f3c896313)
Kiali Server v1.48.0 (62e1107ac53823243fb25ccf971c9109ed906264)
Kiali Container v1.48.0
Components
Istio 1.13.2
Prometheus 2.31.1
Kubernetes v1.22.8-gke.200
Grafana URL N/A
Jaeger URL N/A

**Comment by jmazzitelli** [comment]
@jshaughn @lucasponce is this that bug that was just recently found/fixed? https://github.com/kiali/kiali/issues/4892

**Comment by eitansuez** [comment]
fwiw, i just downgraded kiali to the version that comes with istio 1.13.2 (version 1.45) and i can now navigate to the service page.  service still shows "no health information".

**Comment by lucasponce** [comment]
Hi @eitansuez the "no health information" perhaps is correct if it doesn't have traffic.
Is the tutorial giving you more problems in GKE?
I don't run it for some time in that platform, but let us know if you find something strange there.

**Comment by eitansuez** [comment]
will do.  i suspected the fact that i had no traffic might be the reason for that 'no health info' notification.  ok, i'll ignore that.  so far so good! i'm proceeding with v1.45, onto the "observe" section. :-)

**Comment by jshaughn** [comment]
@jmazzitelli , yes, I think it is the same as https://github.com/kiali/kiali/issues/4892.

**Comment by hhovsepy** [comment]
Hi @eitansuez , is this issue resolved for you after upgrade?

**Comment by eitansuez** [comment]
hi.  i will check and let you know.  might be a few days until i get around to it. thanks!  / eitan
